### PR TITLE
Updates to PrivateLink topic in Cloud Guide

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

- Updated the example in the VPC endpoint connection testing
- Removed reference to Magento Services engagement

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/project/privatelink-service.html

### CODE REVIEW

File: src/cloud/project/privatelink-service.md [hunk 1]
```diff
@@ -131,22 +131,27 @@ To test the connection to the VPC endpoint service:
    curl telnet://<endpoint-service-dns-url>:<port>/ -vvv
    ```
 
-   For example:
+   Example:
 
-   ```bash
-   curl telnet://vpce-004678b7ff06dfxxx-qmmwuxxx.vpce-svc-0aca4ec0ef530xxxx.us-east-1.vpce.amazonaws.com:8443/ -vvv
+   ```terminal
+   $ curl -v telnet://vpce-007ffnb9qkcnjgult-yfhmywqh.vpce-svc-083cqvm2ta3rxqat5v.us-east-1.vpce.amazonaws.com:80 -vvv
```
Comment (hguthrie) [inline src/cloud/project/privatelink-service.md:137]: Are any of the numbers in the example for a specific project or something we would not want to publish? Also, if they are, might be more enlightening to know where the strings come from?

File: src/cloud/project/privatelink-service.md [hunk 2]
```diff
@@ -131,22 +131,27 @@ To test the connection to the VPC endpoint service:
    curl telnet://<endpoint-service-dns-url>:<port>/ -vvv
```
Comment (meker12) [inline src/cloud/project/privatelink-service.md:131]:
```suggestion
   curl -v telnet://<endpoint-service-dns-url>:<port>/
```

File: src/cloud/project/privatelink-service.md (hunk 1 above)
Comment (meker12) [reply src/cloud/project/privatelink-service.md:137]: > Are any of the numbers in the example for a specific project or something we would not want to publish? Also, if they are, might be more enlightening to know where the strings come from?

Those strings are something else!  I'm not sure I understand your exact question, but here's an attempt:

I replaced the IDs in the VPC service endpoint URL with randomly generated strings. 

In the command template immediately above:  I put placeholders for <endpoint-service-dns-url> and <port>.

**Comment by meker12** [comment]
running tests

File: src/cloud/project/privatelink-service.md [hunk 3]
```diff
@@ -128,25 +128,30 @@ To test the connection to the VPC endpoint service:
 1. Run the following CURL command:
 
    ```bash
-   curl telnet://<endpoint-service-dns-url>:<port>/ -vvv
+   curl -v telnet://<endpoint-service-dns-url>:<port>/
```
Comment (hguthrie) [inline src/cloud/project/privatelink-service.md:131]: 🥇

**Pull request merged**